src/inmanta/protocol/methods_v2.py

```diff
@@ -1618,11 +1618,21 @@ def graphql(query: str) -> Any:  # Actual return type: strawberry.types.executio
     GraphQL endpoint for Inmanta.
     Supports paging, filtering and sorting on certain attributes.
 
-    Available queries:
-        - "environments" - Gets a list of created environments.
-            Supports filter and sorting on "id" and "name".
-        - "notifications" - Gets a list of notifications.
-            Supports filters on "cleared" and "environment"
-            Supports sorting on "created"
```

Comment:
Why remove this ?
Also I can't seem to find "notification" in the sample output of the schema you linked, is this expected ?

Reply:
This information is available on the schema.
> Also I can't seem to find "notification" in the sample output of the schema you linked, is this expected ?
This was done before the notifications where merged. I'll update it